python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -82,10 +92,13 @@ def _data_to_table(col_titles: List[str], col_values: List[List[Any]]):
 class Analyzer(object):
     logger = _logger
 
+    LONG_LOG_DURATION_SEC = 2 * 3600.0
+    HIGH_MEASUREMENT_RATE_HZ = 40.0
+
     def __init__(self, file: Union[FileReader, str], output_dir: str = None, ignore_index: bool = False,
                  prefix: str = '',
                  time_range: Tuple[Union[float, Timestamp], Union[float, Timestamp]] = None,
-                 absolute_time: bool = False,
+                 absolute_time: bool = False, truncate_long_logs: bool = True,
```

Comment:
this seems strange to be true by default?

Reply:
I can change to false by default but that would conflict with the command line behavior. I intentionally made that true by default since most of the time I expect someone running a long log for the first time is likely not even interested in these slower plots, so it's probably better to print a warning and plot stuff quickly than to sit for a long time waiting for stuff you aren't interested in.